app/routes/profile/education.js

```diff
@@ -0,0 +1,7 @@
+import { setupFlaggedRoute } from './../../lib/flag-route';
+import OriginalRoute from './old-education';
+
+export default setupFlaggedRoute(OriginalRoute, {
+  flagKey: 'education',
+  enabledRouteName: 'profile/new-education'
```

Comment:
is it `profile/new-education` pointing to new route file? Would it work across two different engines/addons?